src/components/ItemDetail/ItemDetailMetadata.js

```diff
@@ -1,23 +1,83 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import LineEntry from './LineEntry';
+import MultiMetadata from './MultiMetadata';
+import SingleMetadata from './SingleMetadata';
 
 const ItemDetailMetadata = props => {
   if (!props.item) {
     return [];
   }
-  console.log('Props: ' + props);
 
-  const { title } = props.item;
+  const {
+    title = null,
+    abstract = null,
+    caption = null,
+    contributor = null,
+    date = null,
+    description = null,
+    admin_set = null, // division
+    keyword = null,
+    language = null,
+    location = null,
+    provenance = null,
+    publisher = null,
+    related_url = null,
+    rights_holder = null,
+    source = null,
+    subject = null,
+    rights_statement = null,
+    extra_fields = null
+  } = props.item;
+
+  let getCreators = contributors => {
+    let creators = [];
+    for (let entry of contributors) {
+      if (entry.type === 'creator') creators.push(entry);
+    }
+    return creators;
+  };
+
+  let getContributors = contributors => {
+    let c = [];
+    for (let entry of contributors) {
+      if (entry.type === 'contributor') c.push(entry);
+    }
+    return c;
+  };
```

Comment:
Same as above, I'd make these variable declarations instead of calling a function, then refer to the variable in the `items` prop.   Something like...(assuming contributors is an array).
```
const filteredContributors = contributors.filter(entry => { // logic here });